### DISCUSSION TITLE
Any chance we have 32 bit build, like waterfox?

### DISCUSSION THREAD

**Discussion by jasonliul** [Floorp-Projects/Floorp#468] [Feature Request]
Recently some user of waterfox transfer to floorp, that's good news.
Waterfox just like pale moon, lost its direction to move on.

But unfortunately foorp never have x86 build, which prevent more people try it.
i've mentioned this at the very beginning, now it's time to request again.

**Comment by Ryosuke-Asano** [comment]
We do not currently intend to release a 32-bit version, but how much need is there?

Floorp has clearly found a way forward

**Comment by jasonliul** [comment]
i understand you guys are young generations, and since v11, floorp become a pretty success freeware.
Congrats!

There're still many people use 32 bit windows all over the world, they had reason to feel been ignore, disrespect,even discrimination, is this you guys intend to?

Maybe you could put x86 build in "beta" channel, as a gift for those man still use 32 bit windows.

Keep going on, youny man. 
Firefox famaily lost its passion and vitality for long time, we're glad you guys bring something fresh and alive.

**Reply by Ryosuke-Asano** [reply]
OK.

i will write 32bit build script

**Comment by jasonliul** [comment]
後生畏るべし :)

**Comment by Ryosuke-Asano** [comment]
Hi.

I wrote it. We will add support 32bit for Windows while Microsoft supports Windows10(2025). After Ending support windows 10, We will stop to build 32bit.

Thanks.

**Comment by jasonliul** [comment]
ご苦労様でした.

**Comment by adeii** [comment]
Maybe this can work on your system: 
https://github.com/adeii/supermium-portable/releases/download/F130/floorp-11.19.1-portable-x86-VxKex.7z
